src/main/hook/patch/gfx.c

```diff
@@ -35,13 +35,26 @@ typedef Display *(*XOpenDisplay_t)(const char *display_name);
 typedef XVisualInfo *(*glXChooseVisual_t)(
     Display *dpy, int screen, int *attribList);
 typedef void (*glXSwapBuffers_t)(Display *dpy, GLXDrawable drawable);
+typedef void (*glDrawPixels_t)(GLsizei width, GLsizei height, GLenum format, GLenum type, const GLvoid *pixels);
 
-static bool patch_gfx_initialized;
 static XCreateWindow_t patch_gfx_real_XCreateWindow;
 static XOpenDisplay_t patch_gfx_real_XOpenDisplay;
 static glXChooseVisual_t patch_gfx_real_GlXChooseVisual;
 static glXSwapBuffers_t patch_gfx_real_glXSwapBuffers;
+static glDrawPixels_t patch_gfx_real_glDrawPixels;
+
+static bool patch_gfx_initialized;
 static uint16_t s_frame_limit = 0;
+static uint16_t s_init_display_width = 0;
```

Comment:
Same as in the other PR. I suggest to stick to the existing naming pattern to keep consistency.